interview-questions.md

```diff
@@ -7,44 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: (modle . update) then insert the correct information
 
+Researched answer:db:migrate
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: RESTful routes provide mapping between HTTP verbs, controller actions, and CRUD operations in a database.
```

Comment:
Yup, that is correct. The questions is asking which routes require params. You can look at the list of routes in question 4 and see which routes have an id attached to them.